backends/dpdk/run-dpdk-test.py

```diff
@@ -0,0 +1,357 @@
+#!/usr/bin/env python3
```

Comment:
I suspect much of this file is borrowed from the other run-* scripts. Ideally we should factor out the common parts into a module and put it in tools.

Reply:
I started working on a template in the eBPF back end with run-ebpf-test.py, but never got around to generalizing it to the other back ends. Maybe one day I will get around to porting it.

Reply:
@fruffy thanks, let me know when the template is ready.